clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -1068,7 +1114,7 @@ tools::ParsePICArgs(const ToolChain &ToolChain, const ArgList &Args) {
   }
 
   // AMDGPU-specific defaults for PIC.
-  if (Triple.getArch() == llvm::Triple::amdgcn)
+  if (Triple.isAMDGCN())
```

Comment:
another one for the noise reduction patch against amd-stg-open

Reply:
Removed